README.md

```diff
@@ -1,6 +1,9 @@
 # generator-client
 Multi-language client generators for open rpc apis.
 
+## What is Open RPC?
```

Comment:
We should not be sprinkling descriptions of OpenRPC in every readme. Examples shouldn't have an explanation, nor should this. 

Further, the naming when using title casing for open-rpc is OpenRPC with  no spaces.

Reply:
I think it promotes discovery for things like searching github, and npmjs.org
![image](https://user-images.githubusercontent.com/364566/51937195-d6710400-23be-11e9-9aea-a3561d8896e7.png)

Reply:
Also docker hub has `openrpc`

![image](https://user-images.githubusercontent.com/364566/51937321-336cba00-23bf-11e9-8a09-d009cfab0be0.png)

Reply:
I think we can think for ourselves on this one and promote discoverability, since its not a mono repo. for example where should i start right now if i want to get started? the generator client? the spec? the meta spec? is there a open-rpc/open-rpc repo?